Support/Testing/Travis/before-script.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+##
+## Copyright (c) 2014-present, Facebook, Inc.
+## All rights reserved.
+##
+## This source code is licensed under the University of Illinois/NCSA Open
+## Source License found in the LICENSE file in the root directory of this
+## source tree. An additional grant of patent rights can be found in the
+## PATENTS file in the same directory.
+##
+
+set -eu
+
+if [[ "$TARGET" = "Android-ARM" && -n "${LLDB_TESTS-}" ]]; then
+  /tmp/android-sdk-linux/tools/emulator64-arm -avd test -no-skin -no-audio -no-window &
+
+  adb wait-for-device
```

Comment:
We install adb with apt-get in `install.py`. So the `adb` here should already be the one from that package, no?